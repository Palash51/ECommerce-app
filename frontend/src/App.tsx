import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import Base from './routes'
import store from './reducers/store'

function App() {
  return (
    <Provider store={store}>
    <BrowserRouter>
      <Base />
    </BrowserRouter>
    </Provider>
  );
}

export default App;
