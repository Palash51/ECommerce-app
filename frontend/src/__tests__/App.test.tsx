import React from 'react';
import App from '../App';
import { shallow } from 'enzyme';


describe('App component', () => {

    it('Should match snapshot', () => {
        const tree = shallow(<App />)
        expect(tree).toMatchSnapshot();
    })
    
})

