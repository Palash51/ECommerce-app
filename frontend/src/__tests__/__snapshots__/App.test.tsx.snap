// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Should match snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Base />
  </BrowserRouter>
</Provider>
`;
